src/main/java/nextstep/courses/domain/registration/Students.java

```diff
@@ -0,0 +1,50 @@
+package nextstep.courses.domain.registration;
+
+import nextstep.users.domain.NsUser;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class Students {
+    private final int maxUserCount;
+    private final Set<NsUser> users;
```

Comment:
```suggestion
    private final Set<Student> users;
```

Session과 NsUser 사이의 관계가 만들어 지면서 Student(sessionId, nsUserid)와 같은 객체를 생성하고 목록을 가지는 것이 맞지 않을까?